packages/clay-labels-input-field/src/ClayLabelsInputField.soy

```diff
@@ -0,0 +1,171 @@
+{namespace ClayLabelsInputField}
+
+/**
+ * This renders the component's whole content.
+ */
+{template .render}
+	{@param helpText: string}
+	{@param spritemap: string}
+	{@param? _handleButtonClicked: any}
+	{@param? _handleCloseButtonClick: any}
+	{@param? _handleDropdownItemClick: any}
+	{@param? _handleOnInput: any}
+	{@param? _handleOnKeydown: any}
+	{@param? _removeFocusedLabel: any}
+	{@param? contentRenderer: string}
+	{@param? elementClasses: string}
+	{@param? filteredItems: list<?>}
+	{@param? id: string}
+	{@param? label: string}
+	{@param? selectedLabels: list<?>}
+
+	{let $attributes kind="attributes"}
+		class="from-group
+			{if $elementClasses}
+				{sp}{$elementClasses}
+			{/if}
+		"
+
+		{if $id}
+			id="{$id}"
+		{/if}
+	{/let}
+
+	<div {$attributes}>
+		{if $label}
+			<label>{$label}</label>
+		{/if}
+
+		{call .content}
+			{param _handleButtonClicked: $_handleButtonClicked /}
+			{param _handleCloseButtonClick: $_handleCloseButtonClick /}
+			{param _handleDropdownItemClick: $_handleDropdownItemClick /}
+			{param _handleOnInput: $_handleOnInput /}
+			{param _handleOnKeydown: $_handleOnKeydown /}
+			{param _removeFocusedLabel: $_removeFocusedLabel /}
+			{param contentRenderer: $contentRenderer /}
+			{param filteredItems: $filteredItems /}
+			{param helpText: $helpText /}
+			{param selectedLabels: $selectedLabels /}
+			{param spritemap: $spritemap /}
+		{/call}
+	</div>
+{/template}
+
+{template .content}
+	{@param helpText: string}
+	{@param spritemap: string}
+	{@param? _handleButtonClicked: any}
+	{@param? _handleCloseButtonClick: any}
+	{@param? _handleDropdownItemClick: any}
+	{@param? _handleOnInput: any}
+	{@param? _handleOnKeydown: any}
+	{@param? _removeFocusedLabel: any}
+	{@param? contentRenderer: string}
+	{@param? filteredItems: list<?>}
+	{@param? selectedLabels: list<?>}
```

Comment:
Let's call this `selectedItems`?

Reply:
I had put them as items, but I had decided to change thinking it was not a good option 😅. Referring to items is better. Thanks Chema!

Reply:
👍